configs/newdeploy.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
what are these files (olddeploy.yaml and newdeploy.yaml) for?

Reply:
old deploy is the deployment that has the kube-notation image in it.

so in case someone is presenting the old version with seperate kubenotation image, that's the deployment they can use.

Reply:
Lets remove. If needed, we can always get the older version from Git.